_studio/mfx_lib/shared/src/mfx_h264_encode_vaapi.cpp

```diff
@@ -2789,6 +2837,9 @@ mfxStatus VAAPIEncoder::Execute(
     configBuffers.push_back(m_frameRateId);
     configBuffers.push_back(m_qualityLevelId);
 
+    if (VA_INVALID_ID != m_maxSliceSizeId)
```

Comment:
m_maxSliceSizeId might be VA_INVALID_ID  because of some mistake.
```
if (extOpt2->MaxSliceSize != 0)
    configBuffers.push_back(m_maxSliceSizeId);
```
is better and it keeps the same style.

Reply:
done